django/src/rdwatch/models/site_evaluation.py

```diff
@@ -179,10 +170,6 @@ def bulk_create_from_region_model(
 
         site_evals: list[SiteEvaluation] = []
         with transaction.atomic():
-            region = get_or_create_region(
-                region_feature.properties.region_id, region_feature.parsed_geometry
-            )[0]
-
```

Comment:
Okay, sorry but bear with me on this because it may take a while and I may also be wrong.

So This is being removed because we don't need to create a region each time we create a new site.  The probably is I don't believe we have a spot where we upload the region geometry anymore to the Region model.  This was done here with the region_feature.parse_geometry.  It was done a single time, before iterating over all of the other items in the list.


I went looking and noticed that this `bulk_create_from_region_model` is only a  an endpoint located here: https://github.com/ResonantGeoData/RD-WATCH/blob/model-run-region/django/src/rdwatch/views/model_run.py#L375-L379

This is called when loading groundTruth through the script.  It will use this endpoint and I don't believe there is a way it will create the region anymore:
https://github.com/ResonantGeoData/RD-WATCH/blob/model-run-region/scripts/loadGroundTruth.py#L96

I think just enabling this line will create that new geometry.


SideNote:  I don't know if we want to region-model to be adding all these site evaluations at the same time?  Should it be its own contained thing that just creates the region model for the system instead of creating these Sites as well?

Reply:
Oh yeah you're right, good catch! Fixed